code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -479,6 +479,9 @@
 			var/obj/item/clothing/gloves/G = gloves
 			if(G.siemens_coefficient <= 0)
 				total_coeff -= 0.5
+			var/obj/item/clothing/gloves/color/fyellow/greytide = G
```

Comment:
this is going to typecast ANY glove into greytide gloves and cause a runtime when using non-fyellow gloves isn't it?